targets/targets.json

```diff
@@ -15,6 +15,11 @@
         "default_lib": "std",
         "bootloader_supported": false,
         "static_memory_defines": true,
+        "supported_c_libs": {
+                "arm": ["std"],
+                "gcc_arm": ["std"],
```

Comment:
This should be set to `"gcc_arm": ["std", "small"]` so we don't remove features.